lib/suspenders/generators/production/email_generator.rb

```diff
@@ -4,15 +4,15 @@ module Suspenders
   module Production
     class EmailGenerator < Rails::Generators::Base
       source_root File.expand_path(
-        File.join("..", "..", "..", "templates"),
+        File.join("..", "..", "..", "..", "templates"),
         File.dirname(__FILE__),
       )
 
       def smtp_configuration
         copy_file "smtp.rb", "config/smtp.rb"
 
         prepend_file "config/environments/production.rb",
-          %{require Rails.root.join("config/smtp")\n\n}
+          %{require Rails.root.join("config/smtp")\n}
```

Comment:
We, unfortunately, have a test that ensures our `production.rb` has no vertical whitespace in it.